Library/ViewModels/ManagePledgeViewModelTests.swift

```diff
@@ -241,18 +261,24 @@ internal final class ManagePledgeViewModelTests: TestCase {
     self.vm.inputs.menuButtonTapped()
     self.vm.inputs.menuOptionSelected(with: .updatePledge)
 
-    self.goToUpdatePledgeProject.assertValues([Project.template])
+    self.goToUpdatePledgeProject.assertValues([project])
     self.goToUpdatePledgeReward.assertValues([Reward.template])
   }
 
   func testRewardReceivedViewControllerIsHidden_NoReward_Canceled() {
     let backing = Backing.template
       |> Backing.lens.status .~ .canceled
-
-    let project = Project.template
+      |> Backing.lens.reward .~ Reward.noReward
+
+    let project = Project.cosmicSurgery
+      |> Project.lens.rewards %~ { rewards in
```

Comment:
If we just want to switch out the reward at index `0` we could write this here and in other places as:

```swift
|> Project.lens.rewards .~ ([Reward.noReward] + Project.cosmicSurgery.rewards.suffix(from: 1))
```